lib/Jsrt/ChakraCore.h

```diff
@@ -292,7 +292,13 @@ CHAKRA_API
 /// <param name="value">JavascriptString value</param>
 /// <param name="buffer">Pointer to buffer</param>
 /// <param name="bufferSize">Buffer size</param>
-/// <param name="written">Total number of characters written</param>
+/// <param name="writtenLength">Total number of characters written. This is only
+///                             populated when passed with non-null `buffer`.
+/// </param>
+/// <param name="actualLength">Total number of UTF8 characters present in `value`.
```

Comment:
I think it should be number of UTF8 decoded "bytes" corresponding to code-points present in "value". i will fix that up.